core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -17,23 +40,57 @@ object TypelevelPlugin extends AutoPlugin {
   }
 
   import autoImport._
+  import TypelevelKernelPlugin.autoImport._
   import TypelevelSettingsPlugin.autoImport._
   import GenerativePlugin.autoImport._
   import GitHubActionsPlugin.autoImport._
+  import HeaderPlugin.autoImport._
 
   override def globalSettings = Seq(
     tlFatalWarningsInCi := true
   )
 
   override def buildSettings = Seq(
+    organization := "org.typelevel",
+    organizationName := "Typelevel",
+    startYear := Some(java.time.YearMonth.now().getYear()),
+    licenses += "Apache-2.0" -> url("http://www.apache.org/licenses/"),
```

Comment:
Thoughts? These defaults keep even more boilerplate out of `build.sbt` and are very easy to override anyway.

Reply:
Yes.  It's nice if Typelevel projects pick the same.  This would need to be synced with any prospective g8 template.